ntoskrnl/config/cmapi.c

```diff
@@ -2251,7 +2253,7 @@ CmUnloadKey(IN PCM_KEY_CONTROL_BLOCK Kcb,
         }
         else
         {
-            if (CmpEnumerateOpenSubKeys(Kcb, TRUE, TRUE, TRUE) != 0)
+            if (CmpEnumerateOpenSubKeys(Kcb, RegistryLockedExclusively, TRUE, TRUE) != 0)
```

Comment:
Previously, the first `CmpEnumerateOpenSubKeys` call had FALSE, and the second TRUE.
Now, both will use the same value (whatever is passed in RegistryLockedExclusively)?